static/docs/commands-reference/config.md

```diff
@@ -79,74 +79,82 @@ remote. See `dvc remote` for more information.
 
 The DVC cache is a hidden storage (by default located in the `.dvc/cache`
 directory) for files that are under DVC control, and their different versions.
-(See `dvc cache` and [DVC internal
-files](/doc/user-guide/dvc-files-and-directories) for more details.)
+(See `dvc cache` and
+[DVC internal files](/doc/user-guide/dvc-files-and-directories) for more
+details.)
 
 - `cache.dir` - set/unset cache directory location. A correct value must be
   either an absolute path or a path **relative to the config file location**.
   The default value is `cache`, which resolved relative to the default project
   config location results in `.dvc/cache`.
+
   > See also helper command `dvc cache dir` that properly transform paths
   > relative to the present working directory into relative to the project
   > config file.
 
 - `cache.protected` - makes files in the workspace read-only. Possible values
   are `true` or `false` (default). Run `dvc checkout` for the change go into
-  effect. (It affects only files that are under DVC control.)  
+  effect. (It affects only files that are under DVC control.)
+
   Due to the way DVC handles linking between the data files in the cache and
   their counterparts in the working directory, it's easy to accidentally corrupt
   the cached version of a file by editing or overwriting it. Turning this config
   option on forces you to run `dvc unprotect` before updating a file, providing
-  an additional layer of security to your data.  
+  an additional layer of security to your data.
+
   It's highly recommended to enable this mod when `cache.type` is set to
   `hardlink` or `symlink`.
 
 - `cache.type` - link type that DVC should use to link data files from cache to
   your workspace. Possible values: `reflink`, `symlink`, `hardlink`, `copy` or a
-  combination of those, separated by commas e.g: `reflink,hardlink,copy`.  
+  combination of those, separated by commas e.g: `reflink,hardlink,copy`.
+
   By default, DVC will try `reflink,copy` link types in order to choose the most
   effective of those two. DVC avoids `symlink` and `hardlink` types by default
-  to protect user from accidental cache and repository corruption.  
+  to protect user from accidental cache and repository corruption.
+
   > **Note!** If you manually set `cache.type` to `hardlink` or `symlink`, **you
   > will corrupt the cache** if you modify tracked data files in the workspace.
   > See the `cache.protected` config option above and corresponding
-  > `dvc unprotect` command to modify files safely.  
-  There are pros and cons to different link types. Refer to [Cache File
-  Linking](/docs/user-guide/cache-file-linking) for a full explanation of each
-  one.
+  > `dvc unprotect` command to modify files safely.
+
+  There are pros and cons to different link types. Refer to
+  [File link types](/docs/user-guide/cache-file-linking) for a full explanation
```

Comment:
change the name or the link?

Reply:
Do you mean "File link types" or `cache-file-linking` ?

Reply:
I mean - should we do something like [File link types](/doc/user-guide/cache-file-linking#file-link-types-for-the-dvc-cache) or keep it as is - I'm not sure to be honest. Even though neither link, nor title match the content of the link it does not necessary mean that we should change it. So, up to you to decide on this - thus the ? mark at my comment :)

Reply:
I think adding the #anchor is best. I didn't do it because it goes over 80 columns in the .md file but that's OK right? I'll do the change in all the docs where it applies (its a few).

Reply:
Done